### PR TITLE
payload updates for release 1.1.0

### PR DESCRIPTION
**Pull request by moinmoin-sh**
more information can be found here : 
https://github.com/OXRS-IO/OXRS-IO-TouchPanel-ESP32-FW/pull/13
https://github.com/OXRS-IO/OXRS-IO-TouchPanel-ESP32-FW/pull/11

### CODE REVIEW

File: docs/docs/firmware/tp32.md [hunk 1]
```diff
@@ -1085,12 +1085,15 @@ Explanation text goes here...
...
-      "zoom": <number>,
-      "posOffset":[
-        <x number>,
-        <y number>
-      ]
+      "bgImage": {
+        "base64": "<encodeBase64(.png)>",
+        "zoom": <number>,
+        "angle": <number>,
+        "posOffset":[
```
Comment (sumnerboy12) [inline docs/docs/firmware/tp32.md:1092]: Is this now `offset`?

Comment (moinmoin-sh) [reply docs/docs/firmware/tp32.md:1092]: Sorry I can't follow? Do you mean the sequence of keys?

Comment (sumnerboy12) [reply docs/docs/firmware/tp32.md:1092]: I thought you renamed `posOffset` -> `offset` in that latest change?

Comment (moinmoin-sh) [reply docs/docs/firmware/tp32.md:1092]: can't recall ..., The code says `posOffset `

Comment (sumnerboy12) [reply docs/docs/firmware/tp32.md:1092]: This is from the description in your PR...

```json
{
  "tiles":[
    {
      "screen": <number>,
      "tile": <number>,
      "bgImage": 
      {
        "base64": "<encodeBase64(.png)>",
        "zoom": <number>,
        "angle": <number>
        "offset": [<x number>,<y number>],
      }
    }
  ]
}
```

Comment (sumnerboy12) [reply docs/docs/firmware/tp32.md:1092]: Ah but the code doesn't match...

```cpp
tile->alignBgImage(json["bgImage"]["zoom"], json["bgImage"]["posOffset"][0], json["bgImage"]["posOffset"][1], json["bgImage"]["angle"]);
```

Comment (sumnerboy12) [reply docs/docs/firmware/tp32.md:1092]: I prefer `offset` but up to you whether you want to change

Comment (moinmoin-sh) [reply docs/docs/firmware/tp32.md:1092]: I surely overlook that ..., I'll add it to the next release

Comment (sumnerboy12) [reply docs/docs/firmware/tp32.md:1092]: PR pushed to update to `offset` :)

**Pull request merged**